x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/alert_suppression_technical_preview_badge.tsx

```diff
@@ -7,20 +7,30 @@
 
 import React from 'react';
 import { EuiIcon, EuiToolTip } from '@elastic/eui';
+import type { Type } from '@kbn/securitysolution-io-ts-alerting-types';
+import { isSuppressionRuleInGA } from '../../../../../common/detection_engine/utils';
 
 import { TechnicalPreviewBadge } from '../../../../common/components/technical_preview_badge';
 import { useUpsellingMessage } from '../../../../common/hooks/use_upselling';
 
 interface TechnicalPreviewBadgeProps {
   label: string;
+  ruleType: Type | undefined;
 }
 
-export const AlertSuppressionTechnicalPreviewBadge = ({ label }: TechnicalPreviewBadgeProps) => {
+export const AlertSuppressionTechnicalPreviewBadge = ({
```

Comment:
Since we use this for both technical preview and general cases, maybe we should also rename the class to something more general, like `AlertSuppressionBadge`

Reply:
renamed to `AlertSuppressionLabel`, since it's possible it won't have any badge at all